frontends/web/src/locales/en/app.json

```diff
@@ -600,6 +600,19 @@
       "title": "How to restore a wallet from a backup"
     }
   },
+  "success": {
+    "create": {
+      "title": "Your BitBox is ready to go!",
+      "summary": "Here is a summary of what you have done",
+      "info1": "Your wallet has been securely backed up to the micro SD Card. Remove it and keep it safe.",
+      "info2": "You have created a secure device password that unlocks the BitBox.",
+      "info3": "You have created a secure recovery password for your wallet that unlocks your funds and restores your backups."
+    },
+    "restore": {
+      "title": "Your BitBox is ready to go!",
+      "summary": "You have successfully restored a wallet from your backup."
```

Comment:
rename the key also in the japanese translation, so it does not have to be retranslated.

Reply:
done